helm/boring-registry/values.yaml

```diff
@@ -116,3 +116,16 @@ server:
     #   prefix: ""
     #   saEmail: ""
     #   signedURL: false
+
+# the nginx caching proxy configuration
+cachingProxy:
+  enabled: false
+  image:
+    repository: nginx
+    tag: 1.27
+    pullPolicy: IfNotPresent
+  cache:
+    hit: 15m # 200 302
+    download: 5m # 204 307; BR uses 5 minute validity for signed URLs by default
```

Comment:
I could make the caching of download URLs optional.
If a cache time is define - do it, else they will be requested from the BR every time.
WDYT, @odise?

Reply:
Answering myself - keep it simple. If someone needs this setup they should be able to adjust the config easily enough